tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -75,8 +75,14 @@ def test_shiftTemplateDuplicateForm_duplicateSingleTemplate(self):
             follow=True,
         )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(ShiftTemplate.objects.count(), 2)
+        self.assertEqual(2, ShiftTemplate.objects.count())
         for template in ShiftTemplate.objects.all():
+            self.assertEqual(
```

Comment:
We need to test all the fields that are copied in https://github.com/SuperCoopBerlin/tapir/blob/903ddb91484fe14a95528e2b2517317395f9704d/tapir/shifts/views/management.py#L236-L246 to make sure none is forgotten.